packages/astro/src/runtime/client/dev-overlay/overlay.ts

```diff
@@ -374,7 +401,7 @@ export class AstroDevOverlay extends HTMLElement {
 	getPluginTemplate(plugin: DevOverlayPlugin) {
 		return `<button class="item" data-plugin-id="${plugin.id}">
 				<div class="icon">${this.getPluginIcon(plugin.icon)}<div class="notification"></div></div>
-				<span class="sr-only">${plugin.name}</span>
```

Comment:
The tooltip replaces the screenreader only element here. I tried it out with VoiceOver and it has the same result.